src/components/views/rooms/EventTile.tsx

```diff
@@ -1197,8 +1197,8 @@ export class UnwrappedEventTile extends React.Component<EventTileProps, IState>
                             <a href={permalink} onClick={this.onPermalinkClicked}>
                                 {timestamp}
                             </a>
-                            {msgOption}
                         </div>,
+                        msgOption,
```

Comment:
See: https://github.com/matrix-org/matrix-react-sdk/pull/10335/files#diff-9ff5535291b7c2e655ccd3d02c4f632b6f2dc9c30072514641e31c2a8751c57eR1287 and https://github.com/matrix-org/matrix-react-sdk/pull/10335/files#diff-9ff5535291b7c2e655ccd3d02c4f632b6f2dc9c30072514641e31c2a8751c57eR1398. Both of them are placed outside of `mx_EventTile_line`.

Reply:
I didn't understand what these links were supposed to tell me. For the benefit of future reviewers:

This brings `TimelineRenderingType.Thread` in line with the other timeline rendering types by moving the `msgOption` (ie, the read receipt group) outside the `mx_EventTile_line`.